lms/envs/common.py

```diff
@@ -4303,7 +4303,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 CREDIT_TASK_MAX_RETRIES = 5
 
 # Dummy secret key for dev/test
-SECRET_KEY = 'dev key'
+SECRET_KEY = 'dev-key'
```

Comment:
this change isn't a blocker, it will be reverted.

Reply:
Does PyJWT forbid the underscore for some reason?

Reply:
Made this change early in the process but will be reverted in the next commit as it isn't needed anymore. PyJWT doesn't base64 encode if we create PyJWK and then use that for signing the token, so added that manually here [in this line](https://github.com/openedx/edx-platform/pull/31829/files#diff-5d7c4d284e48a3fb0abef394286cf52b7eb37b58a3f66072c169e049d271a914R259)

Reply:
Ah, I see. Hmm. So we won't need to change `SECRET_KEY` in configs, right?